scripts/capdebuginfo

```diff
@@ -0,0 +1,294 @@
+#!/bin/bash
+
+###############################################################################
+# Name: Capture/Debug Information Tool
+# Description:
+#  Script to capture debug info from a MEL user's host
+#
+#
+# Copyright 2007-2013 Mentor Graphics Corporation
+#
+# This file is licensed under the terms of the GNU General Public License
+# version 2.  This program  is licensed "as is" without any warranty of any
+# kind, whether express or implied.
+
+# Variables
+unset BUILDERR   # Path and filename of the MEL build error log file
+unset MELDIR     # Root path of the MEL installation directory
+
+WORKDIR=${PWD}
+HOMEDIR=${HOME}
+HOSTINFOFILE="hostinfo.txt"    # File containing info captured from user's host
+OUTPUTDIRNAME="capdebuginfo"   # Directory containing all info captured by script
+SCRIPT_NAME="$(basename ${BASH_SOURCE})"
+MGLS_LICENSE="MGLS_LICENSE_FILE"
+LM_LICENSE="LM_LICENSE_FILE"
+
+
+usage () {
+   cat >&2 <<END
+
+Description:
+   Capture debug info from a MEL user's host such as host environment and project log files.
+   The Script runs only if environment of the shell is already set using setup-environment script.
+
+Modes of operation:
+   With input arguments: ${SCRIPT_NAME} -e <PN>
+     
+   Without input arguments: ${SCRIPT_NAME}
+   
+Usage: ${SCRIPT_NAME} -e <PN>
+   or  ${SCRIPT_NAME}
+   
+   -e <PN>   Specify the name of package causing build failure
+   -h        Show this usage information
+
+END
+}
+
+
+abspath () {
+    readlink -e "${1}"
+}
+setup_environment() {
+if("Ubuntu"=`lsb_release -i -s`) then
+   source "${MELDIR}"/meta-mentor/scripts/setup-ubuntu
```

Comment:
Add a check here if _for some reason_ sourcing fails; I tried this on dogwood, it was unable to source due to path difference, so I got dpkg-query results for all the packages on host.